downloads/security.md

```diff
@@ -8,25 +8,78 @@ image:
+## Connect Sessions
+* [HKG18-507 – Preparing for future security exploit mitigations](http://connect.linaro.org/resource/hkg18/hkg18-507/)
 
-The original security update from Arm, including links to their original kernel and firmware patches, is available on [developer.arm.com](https://developer.arm.com/support/security-update).
+## 96Boards, webcasts etc
+* [Addressing Meltdown and Spectre with Joakim Bech | OpenHours #86](https://youtu.be/FiINvzyO5R8)
+* [Meltdown, Spectre and OP TEE Webcast](https://youtu.be/rGwgOOSwXBY)
 
-Companies needing help with other or earlier version kernels, 2.x or 3.x, should contact [Linaro Developer Services](https://www.linaro.org/services/#developer-services-contact-us) (services@linaro.org). Linaro Developer Services has significant experience in kernel backports and in securing Arm systems; including secure boot, working with TrustZone, porting OP-TEE and working with Trusted Applications.
 
-In addition to the above kernel work, the Linaro [Security Working Group](https://www.linaro.org/core/security/) has put together both a [blog post and webcast](https://www.linaro.org/blog/meltdown-spectre/) on Meltdown and Spectre from the perspective of the OP-TEE community. They go over the CPU features one by one to get a better understanding of the essence of these attacks. There is additional information for OP-TEE available on the project’s [Security Advisories page](https://www.op-tee.org/security-advisories/).
+## Services
+* Companies needing help with other or earlier version kernels, 2.x or 3.x, should contact [Linaro Developer Services](https://www.linaro.org/services/#developer-services-contact-us) (services@linaro.org). Linaro Developer Services has significant experience in kernel backports and in securing Arm systems; including secure boot, working with TrustZone, porting OP-TEE and working with Trusted Applications.
 
-Additional blogs will be published at [www.linaro.org/blog/](https://www.linaro.org/blog/) to update the status of backports and explain any additional developments.
+## Open Source Projects
+#### Arm
+* General Spectre & Meltdown information from Arm, please see: https://developer.arm.com/support/security-update
+* News:
+ * 2018-02-19: SpectrePrime and MeltdownPrime use a different cache effect as side channel (eviction due to SMP cache snooping) but this is not expected to have any impact on software mitigations for existing micro-architectures. 
```

Comment:
This and other places in the lists are not indented correctly, should have `**` and not just `*` I think.